recipes/pyemcee/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pyemcee" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyemcee-{{ version }}.tar.gz
+  sha256: 6176b2ec3bb415b08c6ea7d2b1e093371096d37ea066acb25fe1dd9504dd279a
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.4
+    - numpy >=1.15
+    - scipy >=1.2
+    - matplotlib >=2.2
+  run:
+    - python >=3.4
+    - numpy >=1.15
+    - scipy >=1.2
+    - matplotlib >=2.2
```

Comment:
This should be matplotlib-base

Reply:
@scopatz I changed from **matplotlib** to **matplotlib-base**.